tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -229,9 +239,121 @@ setup() {
     assert_rootfs_count "$node" "$sandbox_id" "1"
 }
 
+function setup_kbs_credentials() {
+    image=$1
+    user=$2
+    password=$3
+
+    if [ "${KBS}" = "false" ]; then
+        skip "Test skipped as KBS not setup"
+    fi
+
+    if [[ -z ${user} || -z ${password} ]]; then
+        # TODO - is ${GITHUB_ACTION} enough to know we are running in CI?
```

Comment:
It should be enough. Alternatively we could check for `CI=true` which should be exported by Github too: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables